service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala

```diff
@@ -47,21 +47,34 @@ class HttpRequestHandler(
     val uri = req.uri()
     val (path, parameters) = HttpUtils.parseUri(uri)
     val msg = HttpUtils.handleRequest(service, path, parameters)
-    val response = msg match {
+    val textType = "text/plain; charset=UTF-8"
+    val jsonType = "application/json; charset=UTF-8"
```

Comment:
```suggestion
    val jsonType = "application/json"
```

https://www.rfc-editor.org/rfc/rfc8259.html

>    Note:  No "charset" parameter is defined for this registration.
>    Adding one really has no effect on compliant recipients.

Reply:
Thanks. I update it.